libraries/AP_HAL_ChibiOS/hwdef/CubePilot-PPPGW/defaults.parm

```diff
@@ -1,5 +1,14 @@
 NET_ENABLED 1
+NET_OPTIONS 1
 
 # enable hw flow control
 UART1_RTSCTS 1
 
+# swap TX and RX
+UART1_OPTIONS 8
```

Comment:
what!?!!?!?! the pins are swapped?!?!?1

Reply:
they are if the device follows the normal pinout for uarts, as the PPPGW needs crossover. If cubepilot does the final device with the pins swapped in hardware then we can change this